src/main/java/core/basesyntax/service/AreaCalculator.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.model;
+
+public interface AreaCalculator {
+    String color = "";
+    
+    void getArea();
```

Comment:
The `getArea` method should return a `double` or `float` representing the area, rather than being `void`. Update the method signature to `double getArea();` to reflect this.